fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/migration/TenantDatabaseUpgradeService.java

```diff
@@ -70,6 +70,9 @@ public TenantDatabaseUpgradeService(final TenantDetailsService detailsService,
     public void afterPropertiesSet() throws Exception {
         if (databaseStateVerifier.isLiquibaseDisabled()) {
             LOG.warn("Liquibase is disabled. Not upgrading any database.");
+            if (!this.fineractProperties.getMode().isWriteEnabled()) {
+                LOG.warn("Liquibase is disabled, because the `FINERACT_MODE_WRITE_ENABLED` environment variable is set to false.");
```

Comment:
Just styling, I think we can leave out the `this`
Plus, probably it makes more sense to show an environment variable name indepdenent message to the users in case it changes in the future; like "Liquibase is disabled because the current instance is configured as a non-write Fineract instance".
What do you think?

Reply:
You're right, I'm gonna change it.